master/usage/ip-pool-manipulation.md

```diff
@@ -0,0 +1,69 @@
+---
+title: IP Pool Manipulation
+---
+The 
+[IP Pools]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool)
+that Calico starts up with can be manipulated and changed.  This document
+describes how to do that using the `calicoctl` utility.
+
+## Retrieving IP Pool information
+
+The current IP Pools can be retrieved with the following commmand.
+
+```shell
+$ calicoctl get ipPool
+CIDR
+192.168.0.0/16
+
+```
+
+To retrieve the information about a specific IP Pool you must specify the pool
+also.  The `-o yaml` will output in a format that can be captured, manipulated,
+and then used to apply the desired changes.
+
+```shell
+calicoctl get ipPool 192.168.0.0/16 -o yaml
```

Comment:
This is going to get a bit nitpicky, but can we move the output of this command from (1.) to here? i.e.
```
$ calicoctl get ipPool 192.168.0.0/16 -o yaml
- apiVersion: v1
 kind: ipPool
 metadata:
   cidr: 192.168.0.0/16
 spec: {}
```